src/card/card-media.jsx

```diff
@@ -47,6 +47,7 @@ const CardMedia = React.createClass({
         verticalAlign: 'top',
         maxWidth: '100%',
         minWidth: '100%',
+        width:'100%'
```

Comment:
This was the only combo of attributes I looked at cross browsers... I can do some basic testing